Cargo.toml

```diff
@@ -99,7 +99,8 @@ libp2p-rendezvous = { version = "0.14.0", path = "protocols/rendezvous" }
 libp2p-request-response = { version = "0.26.0", path = "protocols/request-response" }
 libp2p-server = { version = "0.12.3", path = "misc/server" }
 libp2p-swarm = { version = "0.44.0", path = "swarm" }
-libp2p-swarm-derive = { version = "0.34.0", path = "swarm-derive" }
+# `libp2p-swarm-derive` may not be compatible with different `libp2p-swarm` non-breaking releases. E.g. `libp2p-swarm` might introduce a new enum variant `FromSwarm` (which is `#[non-exhaustive]`) in a non-breaking release. Older versions of `libp2p-swarm-derive` would not forward this enum variant within the `NetworkBehaviour` hierarchy. Thus the version pinning is required. 
+libp2p-swarm-derive = { version = "=0.34.0", path = "swarm-derive" }
```

Comment:
Ideally, we move this comment onto the same line. If the comment is above, sorting the lines alphabetically doesn't work as easily.

Reply:
I moved the comment to the same line.